test/gtest/uct/uct_test.cc

```diff
@@ -170,28 +170,10 @@ uct_test::uct_test() {
 }
 
 uct_test::~uct_test() {
-    ucs_status_t status;
-    uct_md_attr_t md_attr;
-    uct_md_config_t *md_config;
-    uct_md_h md;
-
-    status = uct_md_config_read(GetParam()->component, NULL, NULL, &md_config);
-    EXPECT_TRUE(status == UCS_OK);
-
-    status = uct_md_open(GetParam()->component, GetParam()->md_name.c_str(),
-                         md_config, &md);
-    EXPECT_TRUE(status == UCS_OK);
-    uct_config_release(md_config);
-
-    status = uct_md_query(md, &md_attr);
-
-    if (((md_attr.cap.flags & UCT_MD_FLAG_SOCKADDR)) ||
-        (strcmp(GetParam()->tl_name.c_str(), "sockaddr"))) {
+    if (m_iface_config != NULL) {
```

Comment:
is it initialized to NULL?

Reply:
it is here - https://github.com/openucx/ucx/pull/4521/files#diff-bf4058cb5ee3a28709818a93c446cafaR161